### DISCUSSION TITLE
how we can use api frontend

### DISCUSSION THREAD

**Discussion by webactor724** [spatie/laravel-multitenancy#523] [Q&A]
please show me how i can find tenant with api request

**Comment by HamzaHassanM** [comment]
I would suggest : 
- Send the tenant name or domain throuh request headers 
- Creating new class let's call it `TenantFinder`  which resolve the tenant through the headers and doing the search for tenant in db 
- Then use the newly created `TenantFinder`  class inside `multitenancy.php`  like `'tenant_finder'=> the new TenantFinder class` 

here is and example 

![image](https://github.com/spatie/laravel-multitenancy/assets/62448602/353d4e85-5bf3-4ba9-9c86-35954e205586)
